assets/js/components/settings/SettingsAdmin.js

```diff
@@ -70,7 +70,10 @@ const SettingsAdmin = () => {
 	return (
 		<Row>
 			{ userInputEnabled && (
-				<Fragment>
+				<div className="
+					mdc-layout-grid__cell
+					mdc-layout-grid__cell--span-12
```

Comment:
We should generally be using `Cell` for grid stuff but we need to address this throughout the rest of the component anyways, so we can leave like this for consistency for now.